src/main/java/org/jenkinsci/plugins/workflow/cps/CpsWhitelist.java

```diff
@@ -103,6 +104,9 @@ public boolean permitsStaticMethod(Method method, Object[] args) {
             String driverTo = "org.codehaus.groovy.runtime.";
             if (cn.startsWith(driverFrom)) {
                 try {
+                    if (Modifier.isPrivate(method.getModifiers())) {
+                        return true;
```

Comment:
Would it possibly make sense to have a check [here](https://github.com/cloudbees/groovy-cps/blob/9e35476a21d6baa801775ce7852d11b4e4a0694d/lib/src/main/java/com/cloudbees/groovy/cps/impl/ContinuationGroup.java#L62-L62) if `e` is an instance of `DefaultInvokerProxyEnv`, and if so, do `return inv.invoke(((DefaultInvokerProxyEnv)e).originalEnv, loc, k)` or something along those lines?